### ISSUE TITLE
implicit_return should be disabled

### ISSUE THREAD

**Issue by n-zhukov** [RedMadRobot/RMR-swift-style-guide#21]
As we discussed offline. **implicit_return** rule should be disabled.

In some cases linter could be failed when the rule is enable.
One more case: [Opening Brace false positive when closure is returned without return statement (Swift 5.1)](https://github.com/realm/SwiftLint/issues/2914)

For example:
```swift
/// It is not fine
public var closure: () -> Void { {
        print("hello")
    }
}
```
```swift
/// It is fine
public var closure: () -> Void { 
    return {
        print("hello")
    }
}
```

**Comment by vani2** [comment]
Please submit PR
